apps/web/src/screens/Profile/components/TransactionHistory/HistoryItem.tsx

```diff
@@ -1,42 +1,28 @@
 import type { FC } from 'react';
-import { Image, StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, View } from 'react-native';
 import { getNetworkInfo } from '@walless/app/utils';
 import type { Transaction } from '@walless/core';
-import { shortenAddress } from '@walless/core';
 import {
 	AnimateDirections,
 	BindDirections,
 	Hoverable,
 	modalActions,
 } from '@walless/gui';
-import { ArrowBottomRight, ArrowTopRight } from '@walless/icons';
-import type { IconProps } from '@walless/icons/components/types';
 
+import ItemAddress from './components/ItemAddress';
+import ItemAmount from './components/ItemAmount';
+import ItemTokenIcon from './components/ItemTokenIcon';
```

Comment:
should use exporting by index file of components to make it cleaner

Reply:
i removed this file in coming PR